examples/step-37/step-37.cc

```diff
@@ -841,8 +841,7 @@ namespace Step37
     const unsigned int nlevels = triangulation.n_global_levels();
     mg_matrices.resize(0, nlevels - 1);
 
-    std::set<types::boundary_id> dirichlet_boundary;
-    dirichlet_boundary.insert(0);
+    const std::set<types::boundary_id> dirichlet_boundary = {0};
```

Comment:
Since you're already in this patch, would you mind terribly changing the name for the variable to `dirichlet_boundary_ids`? Because it's really a set of ids, not of boundaries.

Reply:
Not at all - I fixed this here and in step-66.